### ISSUE TITLE
Not a big issue...but ability to add short notes and export via csv will make this tool complete.

### ISSUE THREAD

**Issue by skdutta-ga** [schmich/marinara#346]

**Comment by FrancoisSchnell** [comment]
This would be very helpful to me too, or a tag system to know how long i've worked on such and such,
https://github.com/schmich/marinara/issues/201
Or an automatic csv sync on my gdrive so i can add it myself here.

**Comment by aschwarzer1** [comment]
Agree!! Notes would be nice. At least notes for a day I did 1 tomato -- I tend to forget what happened that day
